sql/mozfun/search/monetized_search/query.sql

```diff
@@ -0,0 +1,149 @@
+/*
+
+Return a boolean value indicating whether the search is monetized
+
+*/
+CREATE OR REPLACE FUNCTION udf.monetized_search(
```

Comment:
This file should be named `udf.sql` and the function should be `search.monetized_search`

Reply:
Hey Sunah, somehow it sits in my spam folder for a while... just updated the file name and function name. I got an error when I tried to update the dag on my terminal though saying `ValueError: Invalid schedule_interval */5 * * * *.` not sure where it came from though. Is this the reason that the tests failed on CircleCI?

Reply:
That doesn't look related to this PR – try updating your local branch now that we've merged in recent commits from `main`